### PR TITLE
Dashboard button change & composer update

### PR DESCRIPTION
**Pull request by Poseidon281**
This changed the overview in the Dashboard to buttons with redirects:

image

### CODE REVIEW

File: lang/en/commands.php
Comment (notAreYouScared) [inline lang/en/commands.php:1]: Please use single quotes to match the other /lang/en files.

Comment (Poseidon281) [reply lang/en/commands.php:1]: This is already fixed by pint in a other commit and i have not changed anything in this file in this PR

File: composer.lock
Comment (Poseidon281) [inline composer.lock:1]: I updated the composer. I have tested it and i dont think this caused any errors. I dont know why it should but it does not

Comment (notAreYouScared) [reply composer.lock:1]: I have done this locally, and all seems well.... So I'm sure its fine <3

File: resources/views/filament/pages/dashboard.blade.php
Comment (notAreYouScared) [inline resources/views/filament/pages/dashboard.blade.php:1]: Makes no sense to have buttons at the top and have the nav bar on the left. 
They should just be, placeholders? or labels.

**Comment by lancepioch** [comment]
We already have the same links on the side, these are just supposed to be a high level overview. Perhaps we could show the badges on the navigation sidebar?

**Pull request closed without merging**